core/templates/dev/head/components/ExplorationSummaryTileDirective.js

```diff
@@ -34,7 +34,8 @@ oppia.directive('explorationSummaryTile', [function() {
       // If this is not null, the new exploration opens in a new window when
       // the summary tile is clicked.
       openInNewWindow: '@openInNewWindow',
-      isCommunityOwned: '&isCommunityOwned'
+      isCommunityOwned: '&isCommunityOwned',
+      mobileCutoffPx: '@mobileCutoffPx'
```

Comment:
Add comment: 

// If the screen width is below the threshold defined here, the mobile version of the summary tile is displayed. This attribute is optional: if it is not specified, it is treated as 0, which means that the desktop version of the summary tile is always displayed.